packages/subgraph/src/mappings/cfav1.ts

```diff
@@ -164,6 +206,15 @@ function endStreamPeriod(
     incrementPeriodRevisionIndex(streamRevision);
 }
 
+function clipDepositNumber(deposit: BigInt, roundingDown: boolean): BigInt {
```

Comment:
Just wondering, do we have this clipping stuff documented somewhere? (I'm guessing not and it's fine but I just want to know...)

Reply:
should be documented in ethereum-contracts, will add a brief comment to point user to it in the file

Reply:
removed function, it was unused